resources.go

```diff
@@ -146,3 +150,70 @@ func printRunGroups(api resourceAPI) error {
 	printResourceTable([]string{"Run Group ID", "Run Group Title"}, rows)
 	return nil
 }
+
+func postRunJUnitReport(c cliContext, runID int) error {
+	// Get the csv file path either and skip uploading if it's not present
+	fileName := c.String("junit-file")
+	if fileName == "" {
+		return nil
+	}
+	api = rainforest.NewClient(c.String("token"), c.Bool("debug"))
+
+	return writeJunit(c, api)
+}
+
+func augmentJunitFileName(junitFile string, rerunAttempt uint) string {
+	if rerunAttempt > 0 {
+		junitFile = fmt.Sprintf("%v.%v", junitFile, rerunAttempt)
+	}
+
+	return junitFile
+}
+
+// write writeJunit fetches and writes a junit.xml file
+func writeJunit(c cliContext, api resourceAPI) error {
+	var runID int
+	var err error
+
+	if runIDArg := c.Args().Get(0); runIDArg != "" {
+		runID, err = strconv.Atoi(runIDArg)
+		if err != nil {
+			return cli.NewExitError(err.Error(), 1)
+		}
+	} else if deprecatedRunIDArg := c.String("run-id"); deprecatedRunIDArg != "" {
+		runID, err = strconv.Atoi(deprecatedRunIDArg)
+		if err != nil {
+			return cli.NewExitError(err.Error(), 1)
+		}
+
+		log.Println("Warning: `run-id` flag is deprecated. Please provide Run ID as an argument.")
```

Comment:
When are we going to finally drop this 😅 This warning was originally added back in [November 2016](https://github.com/rainforestapp/rainforest-cli/commit/a7467957ca55402ae038d5ba375cad343288a5d3) (meaning it's been deprecated since we released `v2.0.0` the following spring).

Reply:
Ok, I'll drop it in this release 😇